### PR TITLE
automatically download reference

### PR DESCRIPTION
**Pull request by dlaehnemann**
This also required adjusting a bunch of functions in `common.smk`, but our tests should catch any issues with that.

P.S.: I think I need a clean new checkout of this repo locally. Dunno where all these master branch merges and old commits come from...

### CODE REVIEW

File: workflow/rules/ref.smk [hunk 1]
```diff
@@ -0,0 +1,13 @@
+rule get_genome:
```
Comment (johanneskoester) [inline workflow/rules/ref.smk:1]: Isn't this supposed to be called get_transcriptome?

**Review by johanneskoester — APPROVED** [review]

**Review by johanneskoester — CHANGES_REQUESTED** [review]
see above

File: workflow/rules/ref.smk (hunk 1 above)
Comment (tedil) [reply workflow/rules/ref.smk:1]: ... and the output `"resources/transcriptome.fasta"`?

Comment (dlaehnemann) [reply workflow/rules/ref.smk]: Oups. Copy-pasta...

**Review by johanneskoester — APPROVED** [review]

**Pull request merged**